5_People/benefits.md

```diff
@@ -17,16 +17,16 @@ In other words, would something boost your productivity, creativity, or help imp
 
 ### Allowance Budget
 
-The current monthly allowance budget as of *January 2021* is:
+The current monthly allowance budget as of *January 2022* is:
 
- * Niteans: _€125 ($150)_
- * Founders: _€255_
+ * Niteans: _€135 ($150)_
```

Comment:
```suggestion
 * Niteans: _€135 ($155)_
```
Since we round €133.32 -> €135 we should also round $153.51 -> $155

Reply:
Good catch!